src/ServicePulse.Host/app/js/views/endpoint_details/endpoint_details.html

```diff
@@ -21,104 +21,216 @@
 </div>
 
 <div class="container">
-    <busy ng-show="loading" message="Loading details"></busy>
+<div class="tabs">
+    <h5 ng-class="{active: showInstancesBreakdown}">
+        <a ng-click="showInstancesBreakdown = true" class="ng-binding">Instances</a>
+    </h5>
+    <h5 ng-class="{active: !showInstancesBreakdown}">
+        <a ng-click="showInstancesBreakdown = false" class="ng-binding">Message Types</a>
+    </h5>
+</div>
 
-    <div ng-show="!loading" class="row box box-no-click">
-        <div class="col-sm-4 no-side-padding">
-            <div class="row box-header">
-                <div class="col-sm-12 no-side-padding">
-                    <p class="lead">Instance Name</p>
+<section ng-show="showInstancesBreakdown">
```

Comment:
I think we might be better with an `ng-if` here so we don't double render markup. Might be slower to switch between the views, but when on a view it will be quicker